Clinical-Genomics/cgweb
# adds support for of 1508:17

This PR adds support for the 1508 OF 

How to test:
0. install on stage on clinical-db, (clinical-api and cgweb)
1. Enter an Balsamic/Mip+balsamic order in the new excel OF 1508:17, it should use bait set and elution buffer  
1. submit in OP

Expected outcome:
Baitset & Elution buffer ended up in LIMS    
Order created in status-db
